edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentCommand.cs

```diff
@@ -41,14 +42,16 @@ public class EdgeDeploymentCommand : ICommand
             IKubernetes client,
             IEnumerable<IModule> modules,
             IRuntimeInfo runtimeInfo,
-            ICombinedConfigProvider<CombinedDockerConfig> combinedConfigProvider)
+            ICombinedConfigProvider<CombinedDockerConfig> combinedConfigProvider,
+            ICombinedConfigProvider<CombinedKubernetesConfig> kubernetesConfigProvider)
         {
             this.deviceNamespace = KubeUtils.SanitizeK8sValue(Preconditions.CheckNonWhiteSpace(deviceNamespace, nameof(deviceNamespace)));
             this.resourceName = Preconditions.CheckNotNull(resourceName, nameof(resourceName));
             this.client = Preconditions.CheckNotNull(client, nameof(client));
             this.modules = Preconditions.CheckNotNull(modules, nameof(modules)).ToList();
             this.runtimeInfo = Preconditions.CheckNotNull(runtimeInfo, nameof(runtimeInfo));
             this.combinedConfigProvider = Preconditions.CheckNotNull(combinedConfigProvider, nameof(combinedConfigProvider));
```

Comment:
```suggestion
            this.dockerConfigProvider = Preconditions.CheckNotNull(dockerConfigProvider, nameof(dockerConfigProvider));
```

Reply:
I deny this type of changes because code has changed